src/liballoc/boxed.rs

```diff
@@ -596,6 +596,40 @@ impl From<Box<str>> for Box<[u8]> {
     }
 }
 
+use core::convert::TryFrom;
+
+#[derive(Debug, Copy, Clone)]
+#[unstable(feature = "boxed_slice_try_from", issue = "0")]
```

Comment:
Needs an issue (if deemed acceptable)

Reply:
Here and below: trait implementations are insta-stable.